static/usr/lib/systemd/system/keep-entanlged-mounts.target

```diff
@@ -0,0 +1,4 @@
+[Unit]
+Description=Keep entangled mounts
+Wants=run-mnt-data.mount
+Wants=run-mnt-ubuntu-seed.mount
```

Comment:
Afaiu this is to make sure these unmount units survive umount target? Why not put these directly as links in shutdow.target.wants?

Reply:
They would need to be symlinks. But they are generated. I suppose we could have symlinks to `/run/systemd/transient/run-mnt-data.mount`... But they could be broken symlinks, which I do not find elegant.